systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeCorsST.java

```diff
@@ -113,7 +114,17 @@ void testCorsForbidden() {
 
     @BeforeAll
     void beforeAll(ExtensionContext extensionContext) {
-        installClusterOperator(extensionContext, NAMESPACE);
+        install = new Install.InstallBuilder()
+            .withExtensionContext(extensionContext)
+            .withClusterOperatorName(Constants.STRIMZI_DEPLOYMENT_NAME)
+            .withNamespace(NAMESPACE)
+            .withWatchingNamespaces(NAMESPACE)
+            .withBindingsNamespaces(Collections.singletonList(NAMESPACE))
+            .withOperationTimeout(Constants.CO_OPERATION_TIMEOUT_DEFAULT)
+            .withReconciliationInterval(Constants.RECONCILIATION_INTERVAL)
```

Comment:
Can we set those by default? If `witchWatchingNamespaces` is not set, use the same value as `withNamespace` have?

Reply:
Yes, we can.